### DISCUSSION TITLE
Config reference should be sorted alphabetically (and allow copy text)

### DISCUSSION THREAD

**Discussion by shantanugadgil** [ghostty-org/ghostty#4076] [Feature Requests, Ideas] [answered]
Although "most used" _might_ make sense to many users, it seems quite counter-intuitive to me.

ref: https://ghostty.org/docs/config/reference

So, I am proposing that it be alphabetically sorted than "sorted by most used options"

Also, side feature request:

Like the anchor to the option, would it be possible to have a "copy text" option in the web page to copy the text of the config option? # lazy :innocent:   

Disclaimer: I use Ghostty very less, and this is only a fly-by feature request!!! (very lowpri)  :slightly_smiling_face:

**Comment by JakeWharton** [comment, marked as answer]
Sort is covered by https://github.com/ghostty-org/ghostty/discussions/3397

**Reply by shantanugadgil** [reply]
Thanks, I'll close the discussion.
